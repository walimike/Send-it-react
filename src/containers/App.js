import React, { Component } from "react";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import LandingPage from "./LandingPage";
import HomeContainer from './HomeContainer';

import "materialize-css/dist/css/materialize.min.css";

class App extends Component {
  render() {
    return (
      <Router>
        <Switch>
          <Route exact path="/" component={LandingPage} />
          <Route exact path="/home" component={HomeContainer} />
        </Switch>
      </Router>
    );
  }
}

export default App;
