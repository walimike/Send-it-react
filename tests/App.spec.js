import React from 'react';
import { shallow } from 'enzyme';

import App from '../src/containers/App';

describe('App', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});

