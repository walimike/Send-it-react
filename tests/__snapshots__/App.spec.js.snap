// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `ShallowWrapper {}`;
